wpengine/faustjs
# Add phpstan to CI workflow

## Tasks


## Description

- Adds phpstan workflow to faustwp plugin.
- Handles lv1 static checks with success.

## Related Issue(s):



## Testing



1. run composer install
2. run composer phpstan.
3. checks should pass

## Screenshots



## Documentation Changes



## Dependant PRs
